spec/ScaleCommonSpec.js

```diff
@@ -0,0 +1,51 @@
+/* globals describe, beforeEach, it, expect */
+var d3scaleCluster = require('../src/index.js');
```

Comment:
Can you delete ScaleSpec.js since this file seems to replace it?

Reply:
Yes, missed out in the merge